backend/src/main/java/turip/regioncategory/service/RegionCategoryService.java

```diff
@@ -0,0 +1,65 @@
+package turip.regioncategory.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import turip.city.domain.City;
+import turip.city.service.CityService;
+import turip.country.domain.Country;
+import turip.country.service.CountryService;
+import turip.regioncategory.controller.dto.response.RegionCategoriesResponse;
+import turip.regioncategory.controller.dto.response.RegionCategoryResponse;
+import turip.regioncategory.domain.DomesticRegionCategory;
+import turip.regioncategory.domain.OverseasRegionCategory;
+
+@Service
+@RequiredArgsConstructor
+public class RegionCategoryService {
+
+    private static final String KOREA_COUNTRY_NAME = "한국";
+    private static final String DOMESTIC_ETC_NAME = "국내 기타";
+    private static final String OVERSEAS_ETC_NAME = "해외 기타";
+    private static final String DOMESTIC_ETC_IMAGE_URL = "https://example.com/default-image.jpg";
+    private static final String OVERSEAS_ETC_IMAGE_URL = "https://example.com/default-image.jpg";
+
+
+    private final CityService cityService;
+    private final CountryService countryService;
+
+    public RegionCategoriesResponse getRegionCategoriesByCountryType(boolean isKorea) {
+        if (isKorea) {
+            return RegionCategoriesResponse.from(getDomesticRegionCategories());
+        }
+
+        return RegionCategoriesResponse.from(getOverseasRegionCategories());
```

Comment:
c: isKorea를 사용하는 이유는 해외와 구분하기 위함인가요? 그럼 isDomestic 혹은 enum을 사용하는 방식은 어떤지 의견드립니다~

```suggestion
   public RegionCategoriesResponse findRegionCategoriesByCountryType(CountryType countryType) {
    if (countryType == CountryType.KOREA) {
        return RegionCategoriesResponse.from(getDomesticRegionCategories());
    }
    return RegionCategoriesResponse.from(getOverseasRegionCategories());
}
```

Reply:
반영 완료